### PR TITLE
WNYC-177 tracking on play, pause, ended

### PR DESCRIPTION
**Pull request by kimlarocca**
https://nypublicradio-digital.atlassian.net/browse/WNYC-177

### CODE REVIEW

File: components/Signup.vue [hunk 1]
```diff
@@ -48,6 +56,7 @@ const closeAll = () => {
         label="Sign up with Google"
         severity="secondary"
         class="center my-3"
+        @click="onSignup('google')"
```
Comment (bonomite) [inline components/Signup.vue]: use these VLoginWithProvider emits to track the proper event: ["submit-click", "submit-error", "submit-success"]

File: components/Signup.vue [hunk 2]
```diff
@@ -56,6 +65,7 @@ const closeAll = () => {
         severity="secondary"
         class="center"
         label="Sign up with Apple"
+        @click="onSignup('apple')"
```
Comment (bonomite) [inline components/Signup.vue]: use these VLoginWithProvider emits to track the proper event: ["submit-click", "submit-error", "submit-success"]

File: components/Signup.vue [hunk 3]
```diff
@@ -66,6 +76,7 @@ const closeAll = () => {
         label="Sign up"
         slug="/home"
         @login-success="closeAll"
+        @click="onSignup('email')"
```
Comment (bonomite) [inline components/Signup.vue]: use these VSignupWithEmail emits to track the proper event: [  'submit-click',  'submit-error',  'submit-success',  'login-success',]

**Review by bonomite — CHANGES_REQUESTED** [review]
check out the comments. The login/signup components have emits you can use instead of a flat "@click"

File: components/Signup.vue (hunk 1 above)
Comment (kimlarocca) [reply components/Signup.vue]: ok done!

File: components/Signup.vue (hunk 3 above)
Comment (kimlarocca) [reply components/Signup.vue]: ok done!

File: components/Signup.vue (hunk 2 above)
Comment (kimlarocca) [reply components/Signup.vue]: ok done!

**Review by bonomite — APPROVED** [review]
cool

**Pull request merged**